connect/api/v1/internal/intelligence/intelligence_rest_client.py

```diff
@@ -1,24 +1,103 @@
-from connect import settings
-from connect.internal.internal_authencation import InternalAuthentication
+from django.conf import settings
 
 import requests
+import json
+
+from connect.api.v1.internal.internal_authencation import InternalAuthentication
 
 
 class IntelligenceRESTClient:
 
     def __init__(self):
-        self.base_url = settings.INTELIGENCE_REST_ENDPOINT
+        self.base_url = settings.INTELLIGENCE_REST_ENDPOINT
         self.authentication_instance = InternalAuthentication()
 
-    def list_organizations(self, user_email: str):
-        result = []
+    def list_organizations(self, user_email):
+        response = requests.get(
+            url=f"{self.base_url}v2/internal/organization/",
+            headers=self.authentication_instance.get_headers(),
+            params={"user_email": user_email}
+        )
+
+        return response.data
+
+    def get_user_organization_permission_role(self, user_email, organization_id):
+        response = requests.get(
+            url=f"{self.base_url}v2/internal/user/permission/",
+            headers=self.authentication_instance.get_headers(),
+            params={"user_email": user_email, "organization_id": organization_id}
+        )
+        return response.data.role
+
+    def create_organization(self, user_email, organization_name):
+        response = requests.post(
+            url=f"{self.base_url}v2/internal//organization/",
+            headers=self.authentication_instance.get_headers(),
+            json=json.dumps({"user_email": user_email, "organization_name": organization_name})
+        )
+        return response.data
 
-        request = requests.get(
-            url=f"{self.base_url}/v2/internal/list-organizations",
+    def delete_organization(self, organization_id, user_email):
+        response = requests.delete(
+            url=f"{self.base_url}v2/internal/{organization_id}/delete-organization/",
             headers=self.authentication_instance.get_headers(),
             params={"user_email": user_email}
         )
+        return response
+
+    def update_organization(self, organization_id, organization_name):
+        response = requests.put(
+            url=f"{self.base_url}v2/internal/{organization_id}/organization/",
+            headers=self.authentication_instance.get_headers(),
+            params={"organization_name": organization_name}
```

Comment:
the parameter is `user_email`

and you need to add a json here, `{"name": organization_name}`